static-site/src/styles/globals.css

```diff
@@ -9,19 +9,7 @@ html {
     box-sizing: border-box;
     margin: 0;
     padding: 0;
-    font-family: 'Lato', 'Helvetica Neue', sans-serif;
+    /* We use nextjs' next/font to manage our primary font (Lato). `var(--lato)` is defined in pages/_document.tsx */
+    font-family: var(--lato), 'Helvetica Neue', sans-serif;
```

Comment:
Nice find! I _think_ this is because we're using `font-family: lato` not `font-family: var(--lato)`. I'll get to this tomorrow.

Reply:
Thanks! Relatedly, I also noticed that `var(--lato)` renders a different weight variant in a few places, like the blog sidebar:

![diff](https://github.com/nextstrain/nextstrain.org/assets/13424970/60ae1947-5355-49c1-8f74-6ba2a9bad7bf)

Reply:
I dug into this a bunch and there are some aspects of fonts I still don't understand but I am happy with the current state.

> Somehow I'm not getting Lato in a few places on the preview build:

This was fixed by using `var(--lato)` in our styled-components theme. Added by fixing up 
[Move static-site to use Next.JS](https://github.com/nextstrain/nextstrain.org/pull/810/commits/0a56fd9d94e2449fb4652bc75c9cdde0281a77d9)

> Relatedly, I also noticed that var(--lato) renders a different weight variant in a few places, like the blog sidebar:

I've written a bit about what font-weights were asking for vs what weights are being used for rendering in [Remove unused font weight 100](https://github.com/nextstrain/nextstrain.org/pull/810/commits/c4182b4af4d5e6fbd33963f4d19e9a43b5e725f7) but I still can't tell you _why_ the blog sidebar entries were rendered as `font-weight: 100` in your screenshot. They appeared like that for me too, but then I refreshed and they changed to 300: 
<img width="521" alt="image" src="https://github.com/nextstrain/nextstrain.org/assets/8350992/50d4da13-b107-4a09-a3c6-7f7b9743b9d4">

Interestingly, there are two `<h6>` styles which apply to the element in question:

https://github.com/nextstrain/nextstrain.org/blob/6eb8f3a3d3906d4b96e96e35878578810f6eacbe/static-site/src/components/Sidebar/index.jsx#L110-L112

https://github.com/nextstrain/nextstrain.org/blob/6eb8f3a3d3906d4b96e96e35878578810f6eacbe/static-site/src/components/layout.jsx#L67

In your screenshot your browser is using the former, while for me my browser is using the latter as higher priority. Weird. 

Anyways, I've removed the request for 100 (aka "hairline", aka "thin") from the codebase, as it's too thin to be useful anywhere and we never explicitly request it. So the situation you encountered shouldn't happen again. (But can you still test for me?!)

Reply:
Wow, nice digging! 

I'm not sure why my browser picks 200 and yours 300. They have the same specificity of parent class + h6. With static CSS files, it would be resolved by order of appearance. styled-components is different with injection at runtime, and there doesn't seem to be explanation on what's resolved with equal specificity. All I can find is an [FAQ entry](https://styled-components.com/docs/faqs#how-can-i-override-styles-with-higher-specificity) on their alternative to `!important` which doesn't explain our browsers' difference in behavior.

Maybe "nearby weight" picks the lower one when there are equidistant ties? I.e. since mine was using 200, it picked 100 instead of 300?

```
100, 200 -> rendered using 100 (lato-hairline)
300      -> rendered using 300 (lato-light)
```

Anyways, it uses 300 now that you've removed 100, so we're all good here.

Reply:
As an aside, I'm all for removing styled components from the frontend where possible. Next.js supports CSS modules out of the box, and I use them in one or two places in this PR, as typical CSS imports aren't allowed except for at the top level. There are also other styling frameworks we can reach for. But to keep this PR limited in scope I explicitly wanted to modify the styling as little as possible.

Reply:
Yeah, they seem really appealing (they're "just" CSS but scoped), but this was my first time using them and I only ported over one or two files. If we ever want to do true SSG they'll be a much nicer approach than styled components. I'll write more about this in a bit, perhaps on the official "move away from gatsby" issue.